<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>800</width><height>600</height></rect>
  </property>
  <property name="windowTitle">
   <string>ุงููุงุฆูุฉ ุงูุฑุฆูุณูุฉ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">

    <item>
     <widget class="QLabel" name="lbl_welcome">
      <property name="text">
       <string>ูุฑุญุจุงูุ ุงููุณุชุฎุฏู</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string>font-size: 20px; font-weight: bold;</string>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <layout class="QVBoxLayout" name="roomsLayout"/>
      </widget>
     </widget>
    </item>

    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="btn_create">
        <property name="text">
         <string>โ ุฅูุดุงุก ุบุฑูุฉ</string>
        </property>
      </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_logout">
        <property name="text">
         <string>๐ช ุชุณุฌูู ุฎุฑูุฌ</string>
        </property>
      </widget>
      </item>
     </layout>
    </item>

    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="btn_friends">
        <property name="text">
         <string>๐ฅ ุงูุฃุตุฏูุงุก</string>
        </property>
      </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add_friend">
        <property name="text">
         <string>โ ุฅุถุงูุฉ ุตุฏูู</string>
        </property>
      </widget>
      </item>
     </layout>
    </item>

    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="btn_settings">
        <property name="text">
         <string>โ๏ธ ุงูุฅุนุฏุงุฏุงุช</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
